scripts/enable-ecs-agent-gpu-support.sh

```diff
@@ -137,6 +144,18 @@ else
         cuda
 fi
 
+if [[ $AMI_TYPE == *"kernel5dot10gpu" ]]; then
```

Comment:
Q. "line 147 - 155" and "line 51- 51 + line 54- 58" are very similar, do we need all both of then or can we refactor them?

Reply:
Good question! The positioning/separation of L51-L58 and L147-L157 needs to remain. This is because the former is for kernel 5.10 GPU AMIs using post-kepler GPUs in non-airgapped regions and the latter is for all kernel 5.10 GPU AMIs.

L51-L52+L55-L57 and L148-L153 can be refactored to use a common function though. I will update this in the next revision - thanks!